package/MDAnalysis/analysis/psa.py

```diff
@@ -1509,31 +1507,19 @@ def run(self, **kwargs):
              ``trajectory[start:stop:step]`` [``None``]
         stop : int
         step : int
-        store : bool
-             if ``True`` then writes :attr:`PSAnalysis.D` to text and
-             compressed npz (numpy) files [``True``]
-
-             .. deprecated:: 0.19.0
-                `store` will be removed together with :meth:`save_results` in 1.0.0.
-
-        filename : str
-             string, filename to save :attr:`PSAnalysis.D`
 
-             .. deprecated:: 0.19.0
-                `filename` will be removed together with :meth:`save_results` in 1.0.0.
+        .. deprecated:: 0.19.0
```

Comment:
Remove this deprecation note – they are gone now, aren't they?

Reply:
Thanks for the clarification here, this got brought up in #2487 but we weren't sure what the appropriate approach was here. Fixed in 4c7f557 and I'll update #2487 accordingly.